utils/src/pickTreeItem/runQuickPickWizard.ts

```diff
@@ -16,7 +16,6 @@ export async function runQuickPickWizard<TPick>(context: types.PickExperienceCon
 
     const wizard = new AzureWizard(wizardContext, {
         hideStepCount: true,
-        showLoadingPrompt: true,
```

Comment:
Do you know where this is being called? I assume it's probably in mostly in the client extension-- we need to remember to make a note in the changelog that we need to explicitly add this property to anywhere calling it.

Or better yet, it might be better to do this to keep current behavior.
`showLoadingPrompt: wizardOptions.showLoadingPrompt ?? true`